beta-src/src/components/svgr-components/ActionIconSelected.js

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+
+const SvgActionIconSelected = ({iconState}) => (
+  <svg
+    width={42}
+    height={42}
+    viewBox="6 3 42 42"
```

Comment:
is this where you were saying it appears to be shifted a little bit? maybe it's supposed to be 0 0 42 42? if not, this is fine.